### PR TITLE
Adds pill packets, and tricordazine pill packets to the first aid kit pouch.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds pill packets, unrefillable 4 use pill bottles. 
Adds them to the first aid kit pouch instead of the near-useless tricord autoinjector.

## Why It's Good For The Game
The first aid kit pouch is nearly unusably bad compared to the combat autoinjector pouch. The autoinjectors are useless after one use and 5u of tricord does nearly nothing to heal anything. This hopefully makes it more useful.

## Changelog
:cl:
add: Added pill packets. Unrefillable pill bottles. Added paracetamol and tricird ones respectively. 
balance: Tricord autoinjector inside the first aid kit pouch replaced with Tricord pill packets.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -19. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Pariah919** [comment]
>You currently have a negative Fix/Feature pull request delta of -19. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.
> 
no u bot

**Comment by Hughgent** [comment]
Can't crush empty pill packets, negative one billion GBP.

```
/obj/item/storage/pill_bottle/packet/attack_self(mob/living/user)
  . = ..()
  if(!contents.len)
    visible_message("[usr.visible_name] crumples the [name] in their fist.")
    icon = 'blah/blah/trash.dmi'
    icon_state = "paper"
    usr.drop_held_item(src)
```

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -397,6 +397,23 @@
...
+	cant_hold = list(/obj/item/reagent_containers/pill) //Nada. Once you take the pills out. They don't come back in.
+	storage_slots = 4
+	max_w_class = 0
+	max_storage_space = 4
+
+/obj/item/storage/pill_bottle/packet/tricordrazine
+	name = "Pill packet"
+	desc = "This packet containts tricordazine pills. Heals all types of damage minorly. Once you take them out they don't go back in. No more than 2 pills."
+	pill_type_to_fill = /obj/item/reagent_containers/pill/tricordrazine
+
```
Comment (psykzz) [inline code/game/objects/items/storage/firstaid.dm:416]:
```suggestion
```

Add this to a `pill_packet.dm` file instead so we can start to break this out.
The file is getting quite large

Comment (Pariah919) [reply code/game/objects/items/storage/firstaid.dm:416]: alright I'll do that.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-20). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**